Firestore/core/src/firebase/firestore/nanopb/nanopb_util.h

```diff
@@ -104,6 +104,11 @@ inline bool SafeReadBoolean(const bool& nanopb_boolean) {
   return absl::bit_cast<int8_t>(nanopb_boolean) != 0;
 }
 
+template <typename T>
+T* _Nonnull MakeArray(pb_size_t count) {
```

Comment:
XCode demands nullability annotations in this header; any idea why this is happening? I can't see anything that would make it treat the file as Objective-C++.